packages/build/src/types/options/netlify_plugin_run_util.ts

```diff
@@ -16,7 +16,7 @@ type NetlifyPluginRunUtilResult = ExecaChildProcess
 export interface NetlifyPluginRunUtil {
   (
     file: string,
-    // eslint-disable-next-line fp/no-arguments -- params are named `arguments` in the docs. Unrelated to JS's `arguments`
+    // @ts-expect-error this is ok
```

Comment:
Can we explain _what_ is ok?

Reply:
I'd "argue" that it's _not_ ok. We should change the name for the very reason this rule exists